source/tools/timemory-avail/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ endif()
 add_executable(timemory-avail ${_EXCLUDE}
     ${CMAKE_CURRENT_LIST_DIR}/timemory-avail.cpp
     ${CMAKE_CURRENT_LIST_DIR}/timemory-avail.hpp)
+set_target_properties(timemory-avail PROPERTIES DEBUG_POSTFIX ${CMAKE_DEBUG_POSTFIX})
```

Comment:
This can be removed if CMAKE_DEBUG_POSTFIX is cached

Reply:
I think I'd rather not have the executables post-fixed if it's not necessary, and especially not for this executable (since it's the main tool for querying information), but at the very least, put quotes around the variable so that when it's not set, CMake doesn't fail with a wrong number of arguments error.

Reply:
I suggest keeping the debug postfix, but it is okay if you want to remove it. If you remove the debug postfix, then the timemory-avail.exe that is present will be the last one installed. (The debug exe will overwrite the release exe.) That is okay as long as you remember to install the release after you install debug. But if you want to guarantee that you have separate executables, then the easy approach is to use a debug postfix.